styles/effects.css

```diff
@@ -36,3 +36,86 @@ div.glowingBackdrop {
       dark:bg-green-700;
   }
 }
+
+.turtle {
```

Comment:
also here you need to use tailwindcss. If you are not 100% fluent with it. the website team can make the change

Reply:
@AugustinMauroy wanted to prototype quickly so I reverted back to my old ways there - if you can convert to tailwind that would be much appreciated.